src/PKSim.Core/Model/ParameterDistributionMetaDataExtensions.cs

```diff
@@ -12,11 +12,11 @@ public static IReadOnlyList<ParameterDistributionMetaData> DefinedFor(this IEnum
          {
             var allNonPreterm = metaDataForParameter.allForGA(CoreConstants.NOT_PRETERM_GESTATIONAL_AGE_IN_WEEKS).ToList();
 
-            if (!isRealPreterm(originData))
+            if (!originData.IsRealPreterm)
```

Comment:
here. using this weird method

Reply:
ok, I think I know why. 
Some parameters do not differ between different GAs (or at least we just don't have enough information to inform the parameter) and we did not want to create N copies of the same parameter for all possible GAs.
And for those parameters DistributionMetaData is only available for GA=40 (="NOT_PRETERM_GESTATIONAL_AGE_IN_WEEKS")
Thus if we have a preterm of GA=40 (which in fact is not really a preterm :)) - we just take all parameter distributions.
If we have a preterm of GA<40: for each parameter we check first if the distribution data **for the given GA** is available. If yes - take it, if no - take the distribution for GA=40.

Reply:
ok I see. Maybe we can make this a bit clearer in the code